composer.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "spryker-eco/coremedia",
+  "type": "library",
+  "description": "CoreMedia module",
+  "license": "proprietary",
+  "require": {
+    "php": ">=7.1",
+    "spryker-shop/shop-cms-slot-extension": "^0.1.0",
+    "spryker/category-storage": "^1.0.0",
+    "spryker/error-handler": "^2.1.3",
+    "spryker/guzzle": "^2.2.0",
+    "spryker/kernel": "^3.30.0",
+    "spryker/money": "^2.6.0",
+    "spryker/price-product": "^2.0.0 || ^4.0.0",
+    "spryker/price-product-storage": "^2.2.0 || ^4.0.0",
+    "spryker/product-storage": "^1.2.0",
+    "spryker/synchronization-behavior": "^1.0.0",
+    "spryker/util-encoding": "^1.0.0 || ^2.0.0"
+  },
+  "require-dev": {
+    "spryker/code-sniffer": "*",
+    "spryker/testify": "*"
+  },
+  "autoload": {
+    "psr-4": {
+      "SprykerEco\\": "src/SprykerEco/",
+      "SprykerEcoTest\\Client\\CoreMedia\\Helper\\": "tests/SprykerEcoTest/Client/CoreMedia/_support/Helper/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "SprykerEcoTest\\": "tests/SprykerEcoTest/"
+    }
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.0.x-dev"
+    }
+  },
+  "config": {
+    "sort-packages": true
+  }
```

Comment:
Can you add composer scripts "cs-check" here?

Reply:
Will be done in other PR.